demo/go.sum

```diff
@@ -357,11 +357,11 @@ github.com/wundergraph/astjson v0.0.0-20250106123708-be463c97e083 h1:8/D7f8gKxTB
 github.com/wundergraph/astjson v0.0.0-20250106123708-be463c97e083/go.mod h1:eOTL6acwctsN4F3b7YE+eE2t8zcJ/doLm9sZzsxxxrE=
 github.com/wundergraph/cosmo/composition-go v0.0.0-20240124120900-5effe48a4a1d h1:NEUrhuqOaTO1dpW8pz2tu6dKbQAqFvgiF/m4NXdzZm0=
 github.com/wundergraph/cosmo/composition-go v0.0.0-20240124120900-5effe48a4a1d/go.mod h1:9I3gPMAlAY+m1/cFL20iN7XHTyuZd3VT5ijccdU/FsI=
+github.com/wundergraph/cosmo/router v0.0.0-20250218093943-e5b2167f65c7 h1:+kwYDe0/Tm10VTTAgGFXoeVHCnQVwPuKsHijtZKMD4Q=
 github.com/wundergraph/cosmo/router v0.0.0-20250218093943-e5b2167f65c7/go.mod h1:fegv/vONTxD6SRweTJyQSBALVvyst1zI6DOhPj1vJsQ=
 github.com/wundergraph/cosmo/router-tests v0.0.0-20250218093943-e5b2167f65c7 h1:xWDcFs4dCLKgu6ZxW8ADeywL5B8vSLLdZEM8htitzDY=
 github.com/wundergraph/cosmo/router-tests v0.0.0-20250218093943-e5b2167f65c7/go.mod h1:M9w5UOrQkB86E4ZYoLFhvl+04U83B4zm5r2TFtobHxc=
-github.com/wundergraph/graphql-go-tools/v2 v2.0.0-rc.156 h1:OFc0aBKNYav0uZb72WjLNkHaE/onc+joeSTxCnZKdlI=
-github.com/wundergraph/graphql-go-tools/v2 v2.0.0-rc.156/go.mod h1:B7eV0Qh8Lop9QzIOQcsvKp3S0ejfC6mgyWoJnI917yQ=
+github.com/wundergraph/graphql-go-tools/v2 v2.0.0-rc.157 h1:2Fc99E76AKChyJke2FyJYcRJyaTaChr9wnSeybia/FY=
 github.com/wundergraph/graphql-go-tools/v2 v2.0.0-rc.157/go.mod h1:B7eV0Qh8Lop9QzIOQcsvKp3S0ejfC6mgyWoJnI917yQ=
```

Comment:
Looks like we're on [v2.0.0-rc.158](https://github.com/wundergraph/graphql-go-tools/releases/tag/v2.0.0-rc.158) now

Reply:
Thanks